java/algorithm/pom.xml

```diff
@@ -50,4 +54,74 @@
 
   <build>
   </build>
+
+  <profiles>
+    <profile>
+      <id>spotless</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>com.diffplug.spotless</groupId>
+            <artifactId>spotless-maven-plugin</artifactId>
+            <version>${spotless.version}</version>
```

Comment:
Shoudn't this be managed at the top level pom.xml in `pluginManagement` if we are planning to enable it for all modules?

Reply:
We are planning to go module by module and probably after adding all we should move it to the top level?